sdk/python/requirements/py3.9-ci-requirements.txt

```diff
@@ -1,6 +1,6 @@
 #
-# This file is autogenerated by pip-compile with python 3.9
-# To update, run:
+# This file is autogenerated by pip-compile with Python 3.10
```

Comment:
seems like you generated this with the wrong python version?

Reply:
Yea, thanks for the catch!